_pages/algorithm-development/languages/python.md

```diff
@@ -349,15 +267,18 @@ This means that if your project looks like this:
  algorithmia.conf
  src /
       __init__.py
-      main_file.py
+      Algorithmia.py
```

Comment:
typo here, this must be `Algorithm.py` for ADK support; we no longer allow for these files to be renamed, and to ensure that we always have an easy entrypoint we decided upon `Algorithm.py`.